src/io/inputs/model-converter/convertorVisitor.cpp

```diff
@@ -234,7 +234,7 @@ class NotImplemented: public std::runtime_error
 static bool isThePortIsRegistered(const std::string& portId,
                                   const std::vector<YmlModel::Port>& ports)
 {
-    for (const auto [id, _]: ports)
+    for (const auto& [id, _]: ports)
```

Comment:
You can use ports | std::views::keys

Reply:
disregard. I thought  ports was a map

Reply:
ports is a vector not a map, we're getting the id of the ports this way